codegen/main.go

```diff
@@ -43,21 +46,25 @@ local M = {}
 function M.create_{{ $classname | pascalToSnake }}(
 	{{- range $propname, $property := $definition.Properties }}
 	{{- $luaType := luaType $property.Type }}
-	{{- $varName := varName $propname $property.Type $property.Ref }}
-	{{ $varName }}, -- '{{ $luaType }}' ({{ $property.Ref | cleanRef | pascalToSnake }}) {{ $property.Description }}
+	{{- $varName := varName  $propname $property.Type $property.Ref }}
+	{{- $varName := $varName | pascalToSnake }}
```

Comment:
Now we should use `pascalToSnake` a bit more often:
<img width="1266" alt="image" src="https://user-images.githubusercontent.com/2209596/97105340-44447f80-16ba-11eb-882a-c3a399356b32.png">
https://github.com/heroiclabs/nakama/commit/0ff7d3581c1ed90c7372e9f399bbd27462d34821#